custom_components/better_thermostat/models/models.py

```diff
@@ -160,16 +160,16 @@ def convert_outbound_states(self, hvac_mode) -> Union[dict, None]:
 					_LOGGER.warning(
 						f"better_thermostat {self.name}: device config expects no system mode, while the device has one. Device system mode will be ignored"
 					)
-				hvac_mode = None
```

Comment:
If TRV is configured to have no system modes, we shouldn't send system modes to them. So why is this a bug?

Reply:
Same as 168